lib/mongo/cursor.rb

```diff
@@ -185,8 +190,12 @@ def exhausted?
     end
 
     def get_more
-      read_with_retry do
+      if @options[:disable_retry]
         process(get_more_operation.execute(@server))
+      else
+        read_with_retry do
```

Comment:
I don't think getmores should be retryed with the same cursor.  The spec says to create a new cursor on a retryable error.

Reply:
Thanks for the explanation.